app/workers/trigger_webhook_with_object_worker.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class TriggerWebhookWithObjectWorker
+  include Sidekiq::Worker
+
+  # Triggers a webhook for a destructive event (e.g. an unfollow),
+  # where the object cannot be queried from the database.
+  # @param event [String] type of the event
+  # @param object [String] event payload serialized with JSON
+  def perform(event, object)
```

Comment:
Can we add at least basic spec coverage for this class showing that `perform` works?